app/frontend/stores/permit-application-store.ts

```diff
@@ -17,7 +17,6 @@ export const PermitApplicationStoreModel = types
       permitApplicationMap: types.map(PermitApplicationModel),
       tablePermitApplications: types.array(types.reference(PermitApplicationModel)),
       currentPermitApplication: types.maybeNull(types.reference(PermitApplicationModel)),
-      isFetchingPermitApplications: types.optional(types.boolean, false),
```

Comment:
should we not do something in the UI rather than takin this out?

Reply:
this was left over from before the index used the search concern
now the createSearchModel provides a similar field isSearching, and so this is no longer necessary